lib/Url/UrlManager.php

```diff
@@ -262,11 +262,13 @@ public static function getByProfileId($profileId)
      */
     public static function resolveUrl(Url $url)
     {
-        $rewriterSuffix = Url::getRewriter()->getSuffix();
-        if ($rewriterSuffix && substr($url->getPath(), -strlen($rewriterSuffix)) !== $rewriterSuffix) {
-            header('HTTP/1.1 301 Moved Permanently');
-            header('Location: '.$url->getPath().$rewriterSuffix.$url->getQuery());
-            exit;
+        if(!\rex::isBackend()) {
+            $rewriterSuffix = Url::getRewriter()->getSuffix();
+            if ($url->getPath() !== "/sitemap.xml" && $rewriterSuffix && substr($url->getPath(), -strlen($rewriterSuffix)) !== $rewriterSuffix) {
```

Comment:
Die `sitemap.xml` dürfte hier gar nicht ankommen, da diese in der htaccess bereits auf `/index.php?rex_yrewrite_func=sitemap`gelegt wird

Reply:
Sie tut es aber. Vermutlich über die boot.php, wo diese Klasse aufgerufen wird. Spannend ist, dass die robots.txt nicht hier ankommt, obwohl sie genauso umgeleitet wird.

Reply:
Ist denn die Zeile bei dir noch in der htaccess vorhanden?

Reply:
Ja. Soll ich sie mal raus nehmen?
```    RewriteRule ^sitemap\.xml$ %{ENV:BASE}/index.php?rex_yrewrite_func=sitemap [NC,L]
    RewriteRule ^robots\.txt$ %{ENV:BASE}/index.php?rex_yrewrite_func=robots [NC,L]```

Reply:
Nein kann den Fehler nachstellen. Habe auch eine Lösung. Daher schließe ich hier mal.